lib/manifest.js

```diff
@@ -290,6 +290,18 @@ internals.manifest = [
         list: true,
         useFilename: internals.passthruOn('name')
     },
+    {
+        place: 'events',
+        method: 'event',
+        signature: ['events'],
+        list: true
+    },
+    {
+        place: 'event-listeners',
```

Comment:
This one looks good!  Only question is if we want all events stuff under a single directory, kinda like we do with auth.  For example, we could choose to use something like `events/listeners` and `events/definitions`.